webapp/package.json

```diff
@@ -28,16 +28,18 @@
     "@ngrx/effects": "^11.1.1",
     "@ngrx/store": "^11.1.1",
     "@ngx-translate/core": "^13.0.0",
+    "@webcomponents/webcomponentsjs": "^2.6.0",
     "bikram-sambat": "^1.7.0",
     "bikram-sambat-bootstrap": "^1.5.0",
     "bootstrap": "^3.4.1",
     "bootstrap-daterangepicker": "^2.1.30",
-    "enketo-core": "4.41.6",
+    "core-js": "^3.18.3",
+    "enketo-core": "^5.18.1",
     "eurodigit": "^3.1.3",
     "font-awesome": "^4.7.0",
     "google-libphonenumber": "3.2.22",
     "gsm": "^0.1.4",
-    "jquery": "3.2.x",
+    "jquery": "^3.6.0",
```

Comment:
:rofl: I could not figure out why I screwed this up since I was paying attention to what was documented on that page then I made these changes.  I am sure I was trying to follow `Make sure the version of webapp/jquery is the same as webapp/enketo-core/jquery.`, but why did I not notice that this would cause issues with select2??  Finally, after re-reading that doc a couple of times, I realized that it states that jquery `5.6.0+` has issues with select2 (so I should be fine with `3.6.0`... :smile: ).  The only catch is that there is no jquery `5.6.0+`, but instead that must be a typo and definitely should be `3.6.0`.  I have opened a PR to fix the docs: https://github.com/medic/cht-docs/pull/573

Also, I will roll this back to `3.5.1`.

Reply:
Okay, so turns out the reason (of course) that I had jquery set to `3.6.0` is because that is what enketo-core is using.  So now I have three options:
1. Use `3.5.1` but also let enketo-core pull in `3.6.0` (and have both packaged in the bundle). Adds `40Kb` to the bundle size.  
2. Use `3.5.1` and manually edit the webapp package-lock file to keep enketo from pulling in `3.6.0` (and hope that does not break anything in enekto).
3. Find a workaround for the select2 issues and just use `3.6.0`.

Currently I am doing option 2 in the branch, but not sure that is the right way to go....

Reply:
Why `3.5.1`? Right now we're using `3.2.1`.  Does select2 work correctly with `3.5.1`? 

For 2, I think that's should already be covered by `resolutions` (https://github.com/medic/cht-core/blob/master/webapp/package.json#L64), otherwise we can specify an alias in webpack config (https://github.com/medic/cht-core/blob/master/webapp/custom-webpack.config.js#L6). 
Manually editing the package-lock file is not a good idea, because it would mean we'd have to tiptoe around that every time we install a new dependency, and that file is just impossible to follow. 

If there was a workaround to have select2 work with new jQuery that would be the ideal, but I'm unsure that's easy. That missing interaction has been there since before I joined (4 years ago) and I always assumed someone _tried_ to find a workaround before going the documentation route.

Reply:
I believe the reason we were using using `3.2.1` was to match the version of jquery that our old enketo-core was using (as directed by the [dependency documentation](https://docs.communityhealthtoolkit.org/core/guides/update-dependencies/#problems)).  select2 should at least not present the _same_ issue with `3.5.1` of jquery based on our dependency documentation and the discussion out on the jquery issue (which all says that the select2 issue was introduced in `3.6.0`...

Definitely agree that manually editing the package-lock is a bad idea.  I will try blanking things out with an alias since updating the `resolutions` did not prevent both jquery versions from getting added to the bundle....

Reply:
Okay, just wanted leave the conclusion of this thread here for the record.  @dianabarsan suggested that we could just add the Webpack alias: `'jquery': __dirname + '/node_modules/jquery'`. I confirmed that, with this alias, Webpack was able to treeshake the extra jquery copy from the enketo-core/node_modules.